site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -597,28 +604,30 @@ public function showGrading() {
                     return $a->getRotatingSection() < $b->getRotatingSection() ? -1 : 1;
                 });
                 //$total = array_sum($this->core->getQueries()->getTotalTeamCountByGradingSections($gradeable_id, $sections, 'rotating_section'));
-                $total = array_sum($this->core->getQueries()->getTotalUserCountByGradingSections($sections, 'rotating_section'));
+                $total = array_sum($this->core->getQueries()->getTotalTeamCountByGradingSections($gradeable_id, $sections, 'rotating_section'));
+               //BUG HERE
```

Comment:
Is there still a bug there?

Reply:
Yes, if the assignment includes teams the progress bar is still buggy. This will be addressed in the second pull request.